### PR TITLE
Implemented tc

### PR DESCRIPTION
**Pull request by a-meth**
Issue #11756

### CODE REVIEW

File: libr/core/cmd_type.c [hunk 1]
```diff
@@ -745,7 +745,10 @@ static int cmd_type(void *data, const char *input) {
 			r_core_cmd0 (core, "k anal/cc/*");
 			break;
 		case 0:
-			r_core_cmd0 (core, "k anal/cc/*~=cc[0]");
```
Comment (radare) [inline libr/core/cmd_type.c]: you are breaking current tc, without changing the help message or providing an alternative command to achieve the listing o f calling conventions. i think there's another issue for this

Comment (radare) [reply libr/core/cmd_type.c]: i think tc is already  in afcl, so i think this PR is fine, but you should fix the help message first

Comment (a-meth) [reply libr/core/cmd_type.c]: Fixed.

File: libr/core/cmd_type.c [hunk 2]
```diff
@@ -745,7 +745,10 @@ static int cmd_type(void *data, const char *input) {
...
+			r_core_cmd0 (core, "tuc");
```
Comment (radare) [inline libr/core/cmd_type.c]: we can probably simplify this  into 1 single line:

r_core_cmd0 (core, "tuc;tsc;ttc;tec");

**Comment by radare** [comment]
Thanks!!

**Comment by XVilka** [comment]
@a-meth can you please add a few tests for tc/tsc/etc?

**Comment by XVilka** [comment]
By the way PR was wrong. See `help_msg_tc()` which is unfixed, also the actual `tc` implementation.

**Comment by a-meth** [comment]
Can I just remove the help_msg_tc()?

**Comment by XVilka** [comment]
I sent a proper fix already: https://github.com/radare/radare2/pull/12451

**Pull request merged**